lib/pbench/server/auth/__init__.py

```diff
@@ -307,7 +328,7 @@ def get_userinfo(self, token: str = None) -> JSON:
 
     def logout(self, refresh_token: str) -> HTTPStatus:
         """
-        The logout endpoint logs out the authenticated user.
+        The logout endpoint log out the authenticated user.
```

Comment:
I think this was correct before with with "logs". Or perhaps "to log out" or "to terminate a login session", or ... 😆

Reply:
Ah! Yes it sounds weird now

Reply:
Not changed???